### PR TITLE
Test drupal container related type inference

### PR DESCRIPTION
**Pull request by brambaud**
Follow up of #220

### CODE REVIEW

File: tests/src/Type/DrupalContainerRelatedTypeInferenceTest.php [hunk 1]
```diff
@@ -7,12 +7,14 @@
 use mglaman\PHPStanDrupal\Tests\AdditionalConfigFilesTrait;
 use PHPStan\Testing\TypeInferenceTestCase;
 
-class DrupalClassResolverDynamicReturnTypeExtensionTest extends TypeInferenceTestCase
+final class DrupalContainerRelatedTypeInferenceTest extends TypeInferenceTestCase
```
Comment (brambaud) [inline tests/src/Type/DrupalContainerRelatedTypeInferenceTest.php:10]: Not sure about the naming. Is it understandable enough? 🤔

**Review by mglaman — COMMENTED** [review]
I appreciate this follow up! @brambaud these have been some of the best contributions from someone so far. It greatly improved the static analysis capabilities.

File: tests/src/Type/DrupalContainerRelatedTypeInferenceTest.php (hunk 1 above)
Comment (brambaud) [reply tests/src/Type/DrupalContainerRelatedTypeInferenceTest.php]: Yep that is exactly that!

**Comment by brambaud** [comment]
Thanks!
If you need some help on specific tasks let me know if there are some with higher priority than others!

**Review by mglaman — APPROVED** [review]

**Comment by mglaman** [comment]
@brambaud will do! I've started opening more issues as folks try to run this against Drupal core for analysis. If anything looks interesting, feel free. There isn't an exact plan

**Pull request merged**